src/lib/falcor/routes/articles/by-page.js

```diff
@@ -1,11 +1,11 @@
 import falcor from 'falcor';
 import _ from 'lodash';
 
-import { getPaginatedArticle } from 'lib/db';
+import { getPaginatedArticle } from './database-calls';
 
 const $ref = falcor.Model.ref;
 
-export default [
+export const articlesByPageRoutes = [
```

Comment:
Consider making this less verbose as the path already indicates that we are dealing with articles and routes.

Reply:
Yeah I guess it's a bit of a taste debate whether you want semi-globally-unique variable names, I guess an argument could be made that we're already not doing unique things in The Gazelle (we have duplicate component names in the `main` and `admin` folders etc.), so maybe I should just call it routes. The nice part of these names is when you import it it it's clear what it is in another file that may import several routes, but I guess we can also just use `import { routes as byPageRoutes }` etc. to clarify that